lib/system/sysstr.nim

```diff
@@ -156,7 +156,6 @@ proc addChar(s: NimString, c: char): NimString =
       result = rawNewStringNoInit(r)
       result.len = s.len
       copyMem(addr result.data[0], unsafeAddr(s.data[0]), s.len+1)
-      result.reserved = r
```

Comment:
Why was this removed?

Reply:
It's set at line 156 by the `rawNewStringNoInit` call (line 53). `copyMem` call at line 156 touches data only, so I don't see the reason to set `reserved` again.

Reply:
Ok.